src/Event/MultiversionManagerEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\multiversion\Event;
+
+use Drupal\Core\Entity\ContentEntityTypeInterface;
+use Drupal\multiversion\MultiversionMigrationInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * MultiversionManagerEvent class.
+ */
+class MultiversionManagerEvent extends Event {
+
+  /**
+   * @var \Drupal\Core\Entity\ContentEntityTypeInterface[]
+   */
+  protected $entityTypes;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct($entity_types, MultiversionMigrationInterface $migration) {
+    $this->entityTypes = $entity_types;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityType($entity_type) {
```

Comment:
Parent class doesn't implement the `getEntityType()` method, `{@inheritdoc}` can't be used here.

Reply:
The parameter should be named `$entity_type_id` and documented accordingly.